pkg/cli/init-templates/.github/workflows/push.yaml

```diff
@@ -0,0 +1,50 @@
+name: Push to Replicate
+
+on:
+  # Workflow dispatch allows you to manually trigger the workflow from GitHub.com
+  # Go to your repo, click "Actions", click "Push to Replicate", click "Run workflow"
+  workflow_dispatch:
+    inputs:
+      model_name:
+        description: 'Enter the model name, like "alice/bunny-detector"'
+        required: true
+  # # Uncomment these lines to trigger the workflow on every push to the main branch
+  # push:
+  #   branches:
+  #     - main
+
+jobs:
+  push_to_replicate:
+    name: Push to Replicate
+
+    # If your model is large, the default GitHub Actions runner may not 
+    # have enough disk space. If you need more space you can set up a 
+    # bigger runner on GitHub.
+    runs-on: ubuntu-latest
+
+    steps:
+      # This action cleans up disk space to make more room for your
+      # model code, weights, etc.
+      - name: Free disk pace
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          tool-cache: false
+          docker-images: false
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # This action installs Docker buildx and Cog (and optionally CUDA)
+      - name: Setup Cog
+        uses: replicate/setup-cog@v2
+        with:
+          # If you set REPLICATE_API_TOKEN in your GitHub repository secrets,
+          # the action will authenticate with Replicate automatically so you
+          # can push your model
+          token: ${{ secrets.REPLICATE_API_TOKEN }}
+        
+      # If you run the action manually using workflow dispatch, you'll enter
+      # the model name manually. If the workflow is run automatically (say on
+      # a push to main), the model name will be derived from Repsoitory Variables.
+      - name: Push to Replicate
+        run: cog push r8.im/${{ inputs.model_name || vars.DEFAULT_MODEL_NAME }}
```

Comment:
if inputs.model_name is required, would this ever hit the `vars.DEFAULT_MODEL_NAME` branch?

Reply:
Good point. I'm simplifying this by making the input optional, getting rid of DEFAULT_MODEL_NAME, and instead falling back to `cog push` if the input is not set.